test-dsl/src/main/groovy/com/smartbear/readyapi4j/dsl/RestRequestDelegate.groovy

```diff
@@ -137,6 +147,12 @@ class RestRequestDelegate {
             return new XPathAssertionDelegate(xpath, assertionBuilders)
         }
 
-    }
+        XQueryAssertionDelegate xQuery(String xQuery) {
+            return new XQueryAssertionDelegate(xQuery, assertionBuilders)
+        }
 
+        ResponseSlaAssertionDelegate responseSLA(int maxResponseTime) {
```

Comment:
responseSLA would make sense to experienced SoapUI users, but to others it will be a bit cryptic.

I would prefer a friendlier name, maybe maxResponseTime